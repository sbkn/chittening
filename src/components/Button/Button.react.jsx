import React from "react";

const Button = (props) => (
	<button type="button" onClick={props.onClick}>
		Click me
	</button>
);

Button.propTypes = {
	onClick: React.PropTypes.func
};

export default Button;
