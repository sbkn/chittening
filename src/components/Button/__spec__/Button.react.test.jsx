import React from "react";
import renderer from "react-test-renderer";
import Button from "../Button.react.jsx";

describe("Button", () => {

	it("should render correctly", () => {

		const tree = renderer.create(
			<Button/>
		).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
