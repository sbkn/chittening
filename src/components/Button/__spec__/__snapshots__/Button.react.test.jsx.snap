// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<button
  onClick={undefined}
  type="button"
>
  Click me
</button>
`;
